extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DataSourceBuildTimeConfig.java

```diff
@@ -16,4 +16,13 @@
     @ConvertWith(DatabaseKindConverter.class)
     public Optional<String> dbKind = Optional.empty();
 
+    /**
+     * Whether this particular data source should be excluded from the health check if
+     * the general health check for data sources is enabled.
+     * <p>
+     * By default, the health check includes all configured data sources (if it is enabled).
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean healthCheckExclude;
```

Comment:
Could we maybe have something like `health.exclude` instead? That looks a bit more in line with `health.enabled`.

Reply:
Done